### PR TITLE
Validate that references are .dacpac files (#119)

### PR DESCRIPTION
**Pull request by jeffrosenberg**

### CODE REVIEW

**Review by jmezach — APPROVED** [review]
LGTM

File: src/DacpacTool/PackageBuilder.cs [hunk 1]
```diff
@@ -39,14 +35,26 @@ public void AddExternalReference(string referenceFile, string externalParts)
...
             if (!File.Exists(referenceFile))
             {
                 throw new ArgumentException($"Unable to find reference file {referenceFile}", nameof(referenceFile));
             }
 
-            Console.WriteLine($"Adding reference to {referenceFile} with external parts {externalParts}");
-            Model.AddReference(referenceFile, externalParts);
+            // Make sure the file is a .dacpac file
+            var fileType = Path.GetExtension(referenceFile);
+            if (fileType != ".dacpac")
```
Comment (ErikEJ) [inline src/DacpacTool/PackageBuilder.cs:54]: Casing?

File: src/DacpacTool/PackageBuilder.cs [hunk 2]
```diff
@@ -39,14 +35,26 @@ public void AddExternalReference(string referenceFile, string externalParts)
...
+            {
+                throw new ArgumentException($"Invalid type {fileType}, was excepting .dacpac", nameof(referenceFile));
```
Comment (ErikEJ) [inline src/DacpacTool/PackageBuilder.cs:56]: "Invalid filetype" ?

**Review by ErikEJ — CHANGES_REQUESTED** [review]

File: src/DacpacTool/PackageBuilder.cs (hunk 2 above)
Comment (jeffrosenberg) [reply src/DacpacTool/PackageBuilder.cs]: Yeah, that's clearer than what I had, changed 😃

File: src/DacpacTool/PackageBuilder.cs (hunk 1 above)
Comment (jeffrosenberg) [reply src/DacpacTool/PackageBuilder.cs]: Added a `ToLower()` to this, hopefully that's what you were getting at

File: src/DacpacTool/PackageBuilder.cs [hunk 3]
```diff
@@ -39,14 +35,26 @@ public void AddExternalReference(string referenceFile, string externalParts)
...
                 throw new ArgumentException($"Unable to find reference file {referenceFile}", nameof(referenceFile));
             }
 
-            Console.WriteLine($"Adding reference to {referenceFile} with external parts {externalParts}");
-            Model.AddReference(referenceFile, externalParts);
+            // Make sure the file is a .dacpac file
+            string fileType = Path.GetExtension(referenceFile);
+            if (fileType.ToLower() != ".dacpac")
+            {
+                throw new ArgumentException($"Invalid filetype {fileType}, was excepting .dacpac", nameof(referenceFile));
```
Comment (ErikEJ) [inline src/DacpacTool/PackageBuilder.cs:56]: expecting ! :-)

Comment (jeffrosenberg) [reply src/DacpacTool/PackageBuilder.cs:56]: dang 😞

Comment (jeffrosenberg) [reply src/DacpacTool/PackageBuilder.cs]: fixed 😃

**Review by ErikEJ — APPROVED** [review]

**Comment by jmezach** [comment]
@jeffrosenberg Thanks for your contribution! I'll release this as 1.11.3 :).

**Pull request merged**